src/SFML/Window/Unix/KeySymToUnicodeMapping.cpp

```diff
@@ -0,0 +1,1399 @@
+        case 0x0ae7: return 0x25aa; // enfilledsqbullet
+        case 0x0ae1: return 0x25ab; // enopensquarebullet
+        case 0x0adb: return 0x25ac; // filledrectbullet
+        case 0x0ae2: return 0x25ad; // openrectbullet
+        case 0x0adf: return 0x25ae; // emfilledrect
+        case 0x0acf: return 0x25af; // emopenrectangle
+        case 0x0ae8: return 0x25b2; // filledtribulletup
+        case 0x0ae3: return 0x25b3; // opentribulletup
+        case 0x0add: return 0x25b6; // filledrighttribullet
+        case 0x0acd: return 0x25b7; // rightopentriangle
+        case 0x0ae9: return 0x25bc; // filledtribulletdown
+        case 0x0ae4: return 0x25bd; // opentribulletdown
+        case 0x0adc: return 0x25c0; // filledlefttribullet
+        case 0x0acc: return 0x25c1; // leftopentriangle
+        case 0x09e0: return 0x25c6; // soliddiamond
+        case 0x0bcf: return 0x25cb; // circle
+        case 0x0ace: return 0x25cb; // emopencircle
+        case 0x0ade: return 0x25cf; // emfilledcircle
+        case 0x0ae0: return 0x25e6; // enopencircbullet
+        case 0x0ae5: return 0x2606; // openstar
+        case 0x0af9: return 0x260e; // telephone
+        case 0x0aca: return 0x2613; // signaturemark
+        case 0x0aea: return 0x261c; // leftpointer
+        case 0x0aeb: return 0x261e; // rightpointer
+        case 0x0af8: return 0x2640; // femalesymbol
+        case 0x0af7: return 0x2642; // malesymbol
+        case 0x0aec: return 0x2663; // club
+        case 0x0aee: return 0x2665; // heart
+        case 0x0aed: return 0x2666; // diamond
+        case 0x0af6: return 0x266d; // musicalflat
+        case 0x0af5: return 0x266f; // musicalsharp
+        case 0x0af3: return 0x2713; // checkmark
+        case 0x0af4: return 0x2717; // ballotcross
+        case 0x0ad9: return 0x271d; // latincross
+        case 0x0af0: return 0x2720; // maltesecross
+        case 0x0abc: return 0x27e8; // leftanglebracket
+        case 0x0abe: return 0x27e9; // rightanglebracket
+        case 0x04a4: return 0x3001; // kana_comma
+        case 0x04a1: return 0x3002; // kana_fullstop
+        case 0x04a2: return 0x300c; // kana_openingbracket
+        case 0x04a3: return 0x300d; // kana_closingbracket
+        case 0xfe5e: return 0x3099; // dead_voiced_sound
+        case 0xfe5f: return 0x309a; // dead_semivoiced_sound
+        case 0x04de: return 0x309b; // voicedsound
+        case 0x04df: return 0x309c; // semivoicedsound
+        case 0x04a7: return 0x30a1; // kana_a
+        case 0x04b1: return 0x30a2; // kana_A
+        case 0x04a8: return 0x30a3; // kana_i
+        case 0x04b2: return 0x30a4; // kana_I
+        case 0x04a9: return 0x30a5; // kana_u
+        case 0x04b3: return 0x30a6; // kana_U
+        case 0x04aa: return 0x30a7; // kana_e
+        case 0x04b4: return 0x30a8; // kana_E
+        case 0x04ab: return 0x30a9; // kana_o
+        case 0x04b5: return 0x30aa; // kana_O
+        case 0x04b6: return 
```

Comment:
GitHub says there is no newline at the end of this file.

Reply:
Is there a reason, why 'magic numbers' are used instead of constants?
At first glance most of them are defined in `X11/keysym.h`.

Reply:
Hmm, indeed. I must have only seen X11/keysym.h, but didn't take a look at keysymdef.h, where all these codes are.
Will redo this giant switch later. :)